.eslintrc.js

```diff
@@ -47,12 +52,14 @@ module.exports = {
         browser: false,
         node: true,
       },
-      plugins: ['node'],
-      extends: ['plugin:node/recommended'],
+      extends: ['plugin:n/recommended'],
+    },
+    {
+      // test files
+      files: ['tests/**/*-test.{js,ts}'],
+      extends: ['plugin:qunit/recommended'],
       rules: {
-        // this can be removed once the following is fixed
-        // https://github.com/mysticatea/eslint-plugin-node/issues/77
-        'node/no-unpublished-require': 'off',
```

Comment:
is this rule ok to go? the issue is still open.
I have no idea what this solves and if it is still needed tbh..

Reply:
I think that with the automatic plugin switch from `eslint-plugin-node` to `eslint-plugin-n`, this would apply? https://github.com/eslint-community/eslint-plugin-n/pull/57

Reply:
possibly, no idea. probably ok